### PR TITLE
Dockerfile.build: switch to fedora:latest

### PR DESCRIPTION
**Pull request by lsm5**
Resolves: gh#174

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@rhatdan @vrothberg

### CODE REVIEW

**Comment by lsm5** [comment]
@vrothberg @rhatdan what did we do on podman/buildah for the docker hub rate limiting issue, or is that yet to be solved?

**Comment by TomSweeneyRedHat** [comment]
@lsm5, @edsantiago worked his registries.conf magic as outlined in this blog:  https://www.redhat.com/sysadmin/container-quayio

**Comment by lsm5** [comment]
> @lsm5, @edsantiago worked his registries.conf magic as outlined in this blog: https://www.redhat.com/sysadmin/container-quayio

So, openshift has a `origin-hello-openshift` on quay.io. Hopefully, that should Just Work:tm:

File: Dockerfile.build [hunk 1]
```diff
@@ -1,9 +1,12 @@
-FROM golang:1.14-buster
+FROM fedora:latest
```
Comment (rhatdan) [inline Dockerfile.build:1]: Could you fully specify this to grab from Fedora rather then get it at Docker.io, should help rate limits.

Comment (lsm5) [reply Dockerfile.build]: Done

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM, but I'd like @vrothberg to weigh in tomorrow before merging.  I know in the past he's preferred to use a particular version tag rather than latest.

**Comment by ashcrow** [comment]
Looks good!

**Comment by lsm5** [comment]
> LGTM, but I'd like @vrothberg to weigh in tomorrow before merging. I know in the past he's preferred to use a particular version tag rather than latest.

Hmm, I'm not opposed to it, but I'd say `fedora:latest` is pretty stable, and this way we'd not need a change every new release. Either way..

**Comment by vrothberg** [comment]
> @vrothberg @rhatdan what did we do on podman/buildah for the docker hub rate limiting issue, or is that yet to be solved?

Yet to be solved.  I want us to tackle that next week when bug sprint is over.  IMHO a sick CI counts as a bug.

> LGTM, but I'd like @vrothberg to weigh in tomorrow before merging. I know in the past he's preferred to use a particular version tag rather than latest.

Thanks!  I prefer using non-latest tags.  Using "latest" is asking for trouble as it's deliberately a moving target.  Not sure if, for instance, the recent glibc-seccomp fart was introduced to latest but I also don't want to find out :^)

@lsm5, could you use f33 instead?

**Comment by lsm5** [comment]
> @lsm5, could you use f33 instead?

Alright, changing ..

**Comment by rhatdan** [comment]
LGTM
Merge once the tests pass.

**Comment by ashcrow** [comment]
> All checks have passed

:tada:

**Review by vrothberg — COMMENTED** [review]
LGTM, thanks :)

**Pull request merged**